### PR TITLE
memory game

### PR DESCRIPTION
**Pull request by melo4yxa77796**
javascript-javascript3-week2

### CODE REVIEW

**Review by vdaguenet — COMMENTED** [review]
The game works well. The move counter showed me how bad I am at this game 😅
However, you forgot to refactor `fetchCardData` to use `aync`/`await`

**Comment by melo4yxa77796** [comment]
I made changes🙂

File: index.js [hunk 1]
```diff
@@ -141,17 +142,17 @@ document.addEventListener("DOMContentLoaded", () => {
...
+    try {
+      const response = await fetch(apiUrl);
+      emojis = await response.json();
+      createGameBoard();
+    } catch (error) {
+      console.error("Error fetching card data:", error);
+    }
   }
 
   fetchCardData();
```
Comment (vdaguenet) [inline index.js]: `fetchCardData` is now an `async` function and needs to be awaited.

**Review by vdaguenet — APPROVED** [review]

File: index.js (hunk 1 above)
Comment (melo4yxa77796) [reply index.js]: Can you checked again please

**Review by vdaguenet — APPROVED** [review]
👍

**Pull request merged**